gen/cloud-config.yaml

```diff
@@ -15,8 +15,6 @@ root:
     content: |
       [Journal]
       MaxLevelConsole=warning
-      RateLimitInterval=1s
```

Comment:
So, in the latest DC/OS version, we're logging so much that we reach the limit of what journald can handle, right? So, we want to fall back to the journald defaults (which probably are tested by the systemd team and journald should be able to cope with).

But isn't the real problem that we are logging way too much? I stumbled the huge amount of log messages even in small clusters as well and it's a pain to rummage through those logs.